pkg/drawio/layout.go

```diff
@@ -700,8 +701,9 @@ func (ly *layoutS) setIconsLocations() {
 			for _, zone := range vpc.(*VpcTreeNode).zones {
 				setZoneIconsLocations(zone)
 			}
-			ly.setVpcIconsLocations(vpc)
+			ly.setIconsLocationsOnTop(vpc)
 		}
+		ly.setIconsLocationsOnTop(cloud)
```

Comment:
Why is this addition required? Why wasn't `setVpcIconsLocations` called from here before?